mesonbuild/envconfig.py

```diff
@@ -207,7 +207,7 @@ def is_windows(self):
         """
         Machine is windows?
         """
-        return self.system == 'windows'
+        return self.system in set('windows', 'mingw')
```

Comment:
you either need to use `set(['a', 'b'])` or `{'a', 'b'}`. Please use the latter :)

Reply:
1) stop me coding blind late at night 2) one day I'll remember the shorthand for set creation!